sdk/src/dlob/DLOB.ts

```diff
@@ -1269,11 +1354,9 @@ export class DLOB {
 		if (triggerAboveList) {
 			for (const node of triggerAboveList.getGenerator()) {
 				if (oraclePrice.gt(node.order.triggerPrice)) {
-					if (isAuctionComplete(node.order, slot)) {
-						nodesToTrigger.push({
-							node: node,
-						});
-					}
+					nodesToTrigger.push({
```

Comment:
trigger orders cant be in auction?

Reply:
before they trigger order couldn't be triggered until the min auction duration had passed, but im changing that